### PR TITLE
sci-libs/brial: upstream bump

### PR DESCRIPTION
**Pull request by kiwifb**
Package-Manager: Portage-3.0.30, Repoman-3.0.3
Signed-off-by: François René Pierre Bissey <frp.bissey@gmail.com>

### CODE REVIEW

File: sci-libs/brial/brial-1.2.11.ebuild [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+	sci-libs/m4ri[png=]"
+RDEPEND="${DEPEND}"
+
+src_configure() {
+	tc-export PKG_CONFIG
+
+	# with-boost-libdir added to deal with some rather quirky setups
+	# see https://github.com/cschwan/sage-on-gentoo/issues/551
+	econf \
+		--with-boost="${EPREFIX}"/usr \
```
Comment (SoapGentoo) [inline sci-libs/brial/brial-1.2.11.ebuild:29]:
```suggestion
		--with-boost="${ESYSROOT}"/usr \
```

File: sci-libs/brial/brial-1.2.11.ebuild [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+		--with-boost-libdir="${EPREFIX}"/usr/$(get_libdir) \
```
Comment (SoapGentoo) [inline sci-libs/brial/brial-1.2.11.ebuild:30]:
```suggestion
		--with-boost-libdir="${ESYSROOT}"/usr/$(get_libdir) \
```

**Comment by kiwifb** [comment]
Maintenance release. Removed stuff deprecated in c++11 and removed in c++17. Updated configure.ac and some m4 files.

File: sci-libs/brial/brial-1.2.11.ebuild [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+		$(use_enable static-libs static)
```
Comment (SoapGentoo) [inline sci-libs/brial/brial-1.2.11.ebuild:31]: is this needed? We don't added `static-libs` unless it's justified?

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

**Comment by SoapGentoo** [comment]
Please switch from repoman to `dev-util/pkgdev`, since we might remove repoman in the future

File: sci-libs/brial/brial-1.2.11.ebuild (hunk 1 above)
Comment (kiwifb) [reply sci-libs/brial/brial-1.2.11.ebuild:29]: `ESYSROOT` is new to me, I missed its introduction in EAPI 7. I will start using it from now on.

File: sci-libs/brial/brial-1.2.11.ebuild (hunk 3 above)
Comment (kiwifb) [reply sci-libs/brial/brial-1.2.11.ebuild:31]: I cannot think of a consumer who would want it :) if the default is now to not have static unless proven useful I'll oblige gladly. I see that `econf` pass `--disable-static` by default, neat simplification.

**Comment by kiwifb** [comment]
Since you mention `pkgdev`, I was going to use it, but it had an issue with my pgp key. I am guessing that's a setting I had for repoman that needs migrating. Would you be so kind as pointing me in the right direction for that.

**Comment by SoapGentoo** [comment]
```sh
$ cat ~/.config/pkgdev/pkgdev.conf 
[DEFAULT]
commit.scan = true
commit.signoff = true
```
that is all that is generally necessary for committing via `pkgdev commit`

**Comment by kiwifb** [comment]
That's interesting, I was going through the gentoo git workflow (which had a solid update since last time I looked - much cleaner and precise too) and it doesn't mention that. But I probably experienced the problem needing GPG_TTY to be set, which is curious because repoman had no issue. The fact that I am working over ssh probably contributes. I'll try for the next commit.

**Comment by kiwifb** [comment]
Note to self, do not mix repoman and pkgdev on the same branch. 
```
fatal: the receiving end does not support --signed push
```
I'll have to get used to that workflow from start to finish. But right now, I'll have to finish it with repoman or start from scratch.

**Comment by kiwifb** [comment]
> Note to self, do not mix repoman and pkgdev on the same branch.
> 
> ```
> fatal: the receiving end does not support --signed push
> ```
> 
> I'll have to get used to that workflow from start to finish. But right now, I'll have to finish it with repoman or start from scratch.

OK, that was self inflicted. Tried something from https://wiki.gentoo.org/wiki/Gentoo_git_workflow that was probably meant for gentoo main git repo.

**Review by SoapGentoo — APPROVED** [review]

**Comment by thesamesam** [comment]
> That's interesting, I was going through the gentoo git workflow (which had a solid update since last time I looked - much cleaner and precise too) and it doesn't mention that. But I probably experienced the problem needing GPG_TTY to be set, which is curious because repoman had no issue. The fact that I am working over ssh probably contributes. I'll try for the next commit.

Thanks! See also https://wiki.gentoo.org/wiki/Pkgdev#git_signing_errors which I've just added.

**Pull request closed without merging**